### DISCUSSION TITLE
Galaxy sm-j737p discontinued?

### DISCUSSION THREAD

**Discussion by cdd100** [phhusson/treble_experimentations#2516] [Q&A]
I have a galaxy j7 refine (2018) and it is arm32 binder64. I understand this architecture was discontinued in android 12. This phone never received an android 10 vendor update, therefore it is still A only. There is no android 10 firmware, so am I stuck with an android 10 GSI instead of 11?

**Comment by phhusson** [comment]
> I have a galaxy j7 refine (2018) and it is arm32 binder64. I understand this architecture was discontinued in android 12.

What is https://github.com/phhusson/treble_experimentations/releases/download/v416/system-squeak-arm32_binder64-ab-vanilla.img.xz then?

You could build an Android 11 arm32_binder64 a-only. I didn't because the number of such devices is incredibly low

**Reply by cdd100** [reply]
Ok. I will give it a try.

**Reply by cdd100** [reply]
How do I build a Rom newer than android 10?
